src/clr-addons/readonly-combobox/readonly-combobox.directive.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2018-2024 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: 'clr-combobox-container[readOnly], clr-datalist-container[readOnly], clr-select-container[readOnly]',
```

Comment:
Since this is capitalized as `readOnly` I assume you want to make this directive opt-in, as opposed to it working just when using the HTML attribute `readonly` ?

If that's the case, please prefix the selector with "clr" -> `clrReadOnly` or maybe even `clrReadOnlyInput` or similar

Reply:
Let's also rename the file and module - it's not just readonly combobox after all.